include/action.h

```diff
@@ -23,6 +23,11 @@ struct action *action_create(const char *action_name);
 bool action_is_valid(struct action *action);
 
 void action_arg_add_str(struct action *action, const char *key, const char *value);
+void action_arg_add_alist(struct action *action, const char *key);
+void action_arg_add_qlist(struct action *action, const char *key);
+
+struct wl_list *action_get_alist(struct action *acton, const char *key);
+struct wl_list *action_get_qlist(struct action *acton, const char *key);
```

Comment:
I'd prefer using more descriptive naming here, somebody new to the code base (or me in a couple of weeks) won't know what `a` and `q` stand for.

Reply:
Naming: discussed on IRC, for now we leave it as is.
Typo: Fixed.